gapis/resolve/memory.go

```diff
@@ -256,69 +256,8 @@ func MemoryAsType(ctx context.Context, p *path.MemoryAsType, rc *path.ResolveCon
 		return nil, err
 	}
 
-	// Check whether the requested pool was ever created.
-	pool, err := s.Memory.Get(memory.PoolID(p.Pool))
-	if err != nil {
-		return nil, &service.ErrDataUnavailable{Reason: messages.ErrInvalidMemoryPool(p.Pool)}
-	}
-	sz := p.Size
-	if sz == 0 {
-		sz = 0xFFFFFFFFFFFFFFFF
-	}
-
-	dec := s.MemoryDecoder(ctx, pool.Slice(memory.Range{
-		Base: p.Address,
-		Size: sz,
-	}))
-
-	e, err := types.GetType(p.Type.TypeIndex)
-	if err != nil {
-		return nil, err
-	}
-
-	nElems := 1
-	elemSize := 0
-	isSlice := false
-	ty := e
```

Comment:
Why do we have this line? I prefer the name "ty" to "e", because "ty" probably means type. But we have the same value stored in two local variables with highly non descriptive names; and that just hurts readability.

Reply:
Fixed in new commit: Address comment #3.